src/libs/ReportUtils.ts

```diff
@@ -6412,6 +6412,7 @@ function buildOptimisticTaskReport(
     description?: string,
     policyID: string = CONST.POLICY.OWNER_EMAIL_FAKE,
     notificationPreference: NotificationPreference = CONST.REPORT.NOTIFICATION_PREFERENCE.HIDDEN,
+    shouldEscapeText?: boolean,
```

Comment:
```suggestion
    shouldEscapeText = true,
```

`parse` of `ExpensiMark` uses `true` as default for `shouldEscapeText`. So, let us default this to `true` and override it only when required.